src/validator.ts

```diff
@@ -98,7 +98,7 @@ export const validations = {
     )),
 
   methodNotOnlyCallToSuper: warning<Method<Linked>>(node =>
-    !(node.body!.sentences.length === 1 && node.body!.sentences[0].kind === 'Super')),
+    !!node.body  && !(node.body.sentences.length === 1 && node.body.sentences[0].kind === 'Super')),
```

Comment:
we could use:
`node.body.sentences[0].is('Super')` instead of `node.body.sentences[0].kind === 'Super'`

Reply:
I don't know much of the implementation to be sure, lets talk about what `'Super'` represents